utils/fixed_decimal/src/decimal.rs

```diff
@@ -1082,13 +1091,94 @@ impl FixedDecimal {
     /// dec.trunc(0);
     /// assert_eq!("1", dec.to_string());
     /// ```
+    #[cfg(not(feature = "experimental"))]
```

Comment:
nit: let's not do this, write a private `trunc_to_increment_inner` function and then call them from both

Reply:
Would doing something like this also work for you?

```rust
pub fn expand(&mut self, position: i16) {
    #[cfg(feature = "experimental")]
    {
        self.expand_to_increment(position, RoundingIncrement::MultiplesOf1)
    }

    #[cfg(not(feature = "experimental"))]
    {
        // Original impl
    }
}
```

Reply:
Basically I don't want us to duplicate the impl

Reply:
I'll assume that's a yes

Reply:
No, that still duplicates the impl, no?

Reply:
But the reason why we're making the methods experimental is to benchmark them, so we need to preserve the original implementation somehow, right?

Reply:
oh, are these impls distinct?

In that case: we should have internal methods for all of the implementations and use cfgs on thin wrappers

basically we should not do a big cfg/cfg-not pairing for this kind of thing, it's really hard to read. The actual meat of the impl should be in its own method

Reply:
That's fair! In that case I'll extract the implementation into an internal method and call that from the public method.